docs/attributes-registry/gen-ai.md

```diff
@@ -19,6 +19,10 @@ This document defines the attributes used to describe telemetry in the context o
 | `gen_ai.request.model`           | string   | The name of the LLM a request is being made to.                                                  | `gpt-4`                                                                 | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `gen_ai.request.temperature`     | double   | The temperature setting for the LLM request.                                                     | `0.0`                                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `gen_ai.request.top_p`           | double   | The top_p sampling setting for the LLM request.                                                  | `1.0`                                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.stop`           | string   | Up to 4 sequences where the API will stop generating further tokens.                             | `1.0`                                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
There's a mismatch in the name between this and the yaml. You should generate this automatically

Reply:
Got it! Will generate and push this back up